server/models/message.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require( 'mongoose' );
+
+const messageSchema = new mongoose.Schema( {
+    text: {
+        type: String,
+        required: true
+    },
+    senderId: {
+        type: mongoose.Types.ObjectId,
+        ref: 'User',
+        required: true
+    },
+    conversationId: {
+        type: mongoose.Types.ObjectId,
+        ref: 'Conversation',
+        required: true
+    },
+
+},
+    {
+        timestamps: true
+    } );
+
+module.exports = Message = mongoose.model( 'message', messageSchema );
```

Comment:
```suggestion
module.exports = Message = mongoose.model( 'Message', messageSchema );
```
PascalCase is the naming convention for model names in JS

Reply:
replace the message with Message